schemas/paths/related-visitors.yaml

```diff
@@ -9,11 +9,11 @@ get:
     Please visit the [Overview](https://dev.fingerprint.com/reference/related-visitors-api) page to learn more about this API.
   parameters:
     - name: visitor_id
-      description: The [visitor ID](https://dev.fingerprint.com/docs/js-agent#visitorid) for which you want to find the other visitor IDs that originated from the same mobile device.
       in: query
       required: true
       schema:
         type: string
+      description: The [visitor ID](https://dev.fingerprint.com/docs/js-agent#visitorid) for which you want to find the other visitor IDs that originated from the same mobile device.
```

Comment:
It seems we are missing the latest updates from this PR: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/165 

Even though this PR does not seem to have any conflicts.